assets/templates/error.html

```diff
@@ -4,12 +4,10 @@
     <meta charset="utf-8">
     <title>{{.Title}}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="{{asset .Domain "/fonts/fonts.css"}}">
-    {{.CozyUI}}
-    {{.ThemeCSS}}
-    <link rel="apple-touch-icon" sizes="180x180" href="{{asset .Domain "/apple-touch-icon.png" .ContextName}}">
-    <link rel="icon" type="image/png" href="{{asset .Domain "/favicon-32x32.png" .ContextName}}" sizes="32x32">
-    <link rel="icon" type="image/png" href="{{asset .Domain "/favicon-16x16.png" .ContextName}}" sizes="16x16">
+    <link rel="stylesheet">
+    <link rel="apple-touch-icon" sizes="180x180">
+    <link rel="icon" type="image/png" sizes="32x32">
+    <link rel="icon" type="image/png" sizes="16x16">
   </head>
```

Comment:
I don't think this file is useful, as dispers's actors are not supposed to provide any UI

Reply:
Understood. I wanted to keep the possibility to provide basic UI. I will delete everything about UI.

Reply:
Did you have a use-case in mind for the need of serving an UI?

Reply:
Absolutely not